configure.ac

```diff
@@ -382,6 +378,13 @@ fi
 AM_CONDITIONAL(ENABLE_DOCS,
     [test "$enable_docs" = "yes"])
 
+AC_ARG_ENABLE([gtest],
+    [AC_HELP_STRING([--enable-gtest],
+                    [build unit tests @<:@default=no@:>@])],
+    [], [enable_gtest="no"])
```

Comment:
Please use **--enable-tests** instead since it's possible we could switch test frameworks at some point in the future.  This also makes it more consistent with libva-utils and intel-vaapi-driver "--enable-tests" option.